touchforms/backend/xformserver.py

```diff
@@ -165,7 +165,7 @@ def handle_request(content, server):
         session_id = content['session-id']
 
     action = content['action']
-    logger.info('Received action %s for session %s' % (action, session_id))
+    logger.info('event=received action=%s session=%s' % (action, session_id))
```

Comment:
good to know. my logging knowledge is pretty elementary. i'll play around with that